components/client/home.tsx

```diff
@@ -0,0 +1,79 @@
+"use client";
+
+import type { ReactElement } from "react";
+import { useState } from "react";
+import BarChart from "@/components/client/bar-chart";
+import Heatmap from "@/components/client/heatmap";
+import OrderGrid from "@/components/client/order-grid";
+import Calendar from "@/components/client/calendar";
+import Grid from "@mui/material/Grid2";
+import IconButton from "@mui/material/IconButton";
+import ArrowBackIosNewIcon from "@mui/icons-material/ArrowBackIosNew";
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+
+const Home = (): ReactElement => {
+  const components = [
+    <BarChart key="first" />,
+    <Heatmap key="second" />,
+    <OrderGrid key="third" />,
+  ];
+
+  const InitialPointer = 0;
+  const MinPointer = 0;
+  const PointerIncrement = 1;
```

Comment:
@Danar435 is this a type or a variable? Variables use camelCase, types PascalCase